app/src/main/java/com/example/mylittlelibrary/data/Book.kt

```diff
@@ -1,5 +1,12 @@
 package com.example.mylittlelibrary.data
-
+import com.google.gson.annotations.SerializedName
 import java.util.*
 
-data class Book(var name : String, var lendTo: String, var date: Date)
+data class Book(@SerializedName("name") var name : String, var lendTo: String, var date: Date)
+{
+    val bookString= name+" "+lendTo
+    override fun toString(): String {
```

Comment:
Add a toString method to the class. URL:https://docs.sonarqube.org/display/SONARQUBE45/Developers%27+Seven+Deadly+Sins

Reply:
Use a Kotlin shortcut: return "$name %lendTo"